src/EthersSafe.ts

```diff
@@ -52,6 +110,24 @@ class EthersSafe implements Safe {
     return txHash
   }
 
+  /**
+   * Signs data using the current owner account.
+   *
+   * @param hash - The data to sign
+   * @returns The Safe signature
+   */
+  async signMessage(hash: string): Promise<SafeSignature> {
```

Comment:
I am not sure about the name as the Safe also has a method called `signMessage`

Reply:
Is `signTransactionHash` better?

Reply:
yes, this would make it a lot clearer for me :)